UCSD/campus-mobile
# Accessibility card menu option change

## Summary
[Accessibility: Card options menu verbiage changes#1627](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1627)


## Changelog

[General] [Change] - Card options menu verbiage changes
 Card option "reload" changed to "reload this card"
 Card option "hide" changed to "hide this card"

## Test Plan
- Turn on the screen reader feature on your mobile device (built-in or installed)
- Open the app, reload and hide cards with screen reader on
- the screen reader should say "reload/hide this card" with no other functional change
